File: spec/javascripts/ponytail/models/column_spec.js
describe("Column", function() {
  describe("#constructor", function() {
    var column;
    beforeEach(function() {
      column = new Column({name: "id", type: "integer"});
    });
    it("should be able to set name", function() {
      expect(column.name).toBe("id");
    });
    it("should be able to set type", function() {
      expect(column.type).toBe("integer");
    });
  });
});

File: spec/models/ponytail/column_spec.rb
require 'spec_helper'

module Ponytail
  describe Column do
    describe 'include Dirty' do
      let(:column) { Column.new name: 'Fate Testarossa', type: :string }
      before { column.name = 'Fate T.Harlaown' }

      it { expect(column).to be_changed }
      it { expect(column.changes[:name]).to eq ['Fate Testarossa', 'Fate T.Harlaown'] }
    end
  end
end
